letsencrypt/crypto_util.py

```diff
@@ -56,7 +68,10 @@ def init_save_key(key_size, key_dir, keyname="key-letsencrypt.pem"):
     with key_f:
         key_f.write(key_pem)
 
-    logger.info("Generating key (%d bits): %s", key_size, key_path)
+    if key_algo == "RSA":
+            logger.info("Generating RSA key (%d bits): %s", rsa_key_size, key_path)
```

Comment:
I think this log function was originally in the wrong place, since it comes after the key is already generated. These should happen before the key is generated (and outside of this function, which is now about saving keys).